lib/labels.js

```diff
@@ -0,0 +1,260 @@
+'use strict';
+
+const https = require('https');
+const querystring = require('querystring');
+
+const request = Symbol('request');
+const getPath = Symbol('getPath');
+
+const parseLink = link => {
+  const parsedLinks = link
+    .split(',')
+    .map(s => s.split(';'))
+    .map(([url, rel]) => [rel.trim().slice(5, -1), url.trim().slice(1, -1)]);
+  return new Map(parsedLinks);
+};
+
+class LabelHandler {
+  constructor(options = {}) {
+    this.options = options;
+  }
+
+  async [request](path, method, data) {
+    const reqOptions = {
+      headers: {
+        'user-agent': this.options.user,
+        authorization: `token ${this.options.token}`,
+        accept: 'application/vnd.github.symmetra-preview+json',
```

Comment:
I also don't like using this but we need it to be able to add a description. Otherwise, it will be only name and color. See https://developer.github.com/v3/issues/labels/#create-a-label and https://developer.github.com/changes/2018-02-22-label-description-search-preview.

Reply:
If that's the case, I think we can avoid using descriptions for now until it becomes available in a stable API.